dlp/snippets/deid_test.py

```diff
@@ -299,6 +299,32 @@ def test_deidentify_with_replace_infotype(capsys):
     assert "My favorite site is [URL]" in out
 
 
+def test_deidentify_with_simple_word_list(capsys):
+    deid.deidentify_with_simple_word_list(
+        GCLOUD_PROJECT,
+        "My Room is rm-green",
+        "CUSTOM_ROOM_ID",
+        ["RM-GREEN", "RM-YELLOW", "RM_ORANGE"],
```

Comment:
Did you use RM_ORANGE intentional? It works fine.

Reply:
did not get this comment.

Reply:
@Avani-Thakker-Crest The other entries use hyphen, Orange uses underscore.

Reply:
ohh yeahhhh, let me update this one.

Reply:
updated, thank you for noticing :)